agents/agents-features/agents-features-sql/build.gradle.kts

```diff
@@ -35,10 +35,10 @@ kotlin {
                 api(libs.exposed.jdbc)
                 api(libs.exposed.json)
                 api(libs.exposed.kotlin.datetime)
-                api(libs.postgresql)
-                api(libs.mysql)
-                api(libs.h2)
-                api(libs.sqlite)
+                compileOnly(libs.h2)
```

Comment:
Shouldn't drivers be `runtimeOnly`? Because we don't use components from these dependencies directly, but these components are loaded at runtime based on the data source type. That's why AFAIK these are usually made `runtimeOnly`

Reply:
Ok, so we're doing this to avoid adding these dependencies forcibly to the end users, aiming for "provided" behavior instead. Makes sense, thanks